packages/expo/src/facebook-ads/index.web.ts

```diff
@@ -0,0 +1,30 @@
+import { UnavailabilityError } from 'expo-errors';
+
+function unavailable(name: string) {
```

Comment:
Not a big deal here since it's easy to read the whole file but prefer using verb phrases for function names (ex: `assertUnavailable`) or just writing the `throw` statement in each method